docs/verification-request.md

```diff
@@ -0,0 +1,83 @@
+# Verification requests
+
+## VerificationRequest
+
+VerificationRequest is that class responsible for transitioning between the various phases that a verification request can be in. This is both for self-verification over to-device messages, as for verification of other users over in-room messages. It also creates the verifier object (depending on the chosen method) once the .start event is received.
```

Comment:
We should put the `VerificationRequest` class in backticks like the other classes (appears other places)

```suggestion
`VerificationRequest` is that class responsible for transitioning between the various phases that a verification request can be in. This is both for self-verification over to-device messages, as for verification of other users over in-room messages. It also creates the verifier object (depending on the chosen method) once the .start event is received.
```